sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -138,6 +152,8 @@ def test_top_key(self):
         ['aa', 'bbb', 'c', 'dddd'] | combine.Top.Of(3, key=len, reverse=True),
         [['c', 'aa', 'bbb']])
 
+  @unittest.skipIf(sys.version_info[0] > 1, 'deprecated comparator')
```

Comment:
Will `sys.version_info[0] > 1` ever evaluate to True? Should we remove this entirely?

Reply:
*evaluate to False. Did you mean > 2?

Reply:
I believe you are right, updated the two uses of it here.

Reply:
Oh, thanks. I was just testing this on Py2.